code/datums/uplink_items/uplink_nuclear.dm

```diff
@@ -517,6 +509,14 @@
 
 // IMPLANTS
 
+/datum/uplink_item/implants/krav_implant
+	name = "Krav Maga Implant"
+	desc = "A biochip that teach you Krav Maga when implanted, great as a cheap backup weapon. Warning, the biochip will override any other fighting styles such as CQC."
+	reference = "KMI"
+	item = /obj/item/implanter/krav_maga
+	cost = 5
```

Comment:
```suggestion
	cost = 25
```
This will need to be adjusted due to the passing of the TC Inflation Act of 2023.

Reply:
Done!